horace_core/sqw/coord_transform/@aProjectionBase/aProjectionBase.m

```diff
@@ -144,6 +148,9 @@
         % methods for specific projections, but 4D transformation is
         % algorithmically simpler so actively used in tests.
         do_3D_transformation_ = true;
+        %if set to true, disables pixel preselection part of cut algorithm.
+        % Cut then works over all pixels present in the source image.
+        disable_pix_preselection_ = false;
```

Comment:
state that by default we expect this var to be false as disabling is inefficient